readme.md

```diff
@@ -863,6 +863,20 @@ echo $faker->bank; // "Volksbank Stuttgart"
 
 ```
 
+### `Faker\Provider\de_DE\Person`
+
+```php
+<?php
+
+echo $faker->title; // "Dr., Prof."
+echo $faker->salutation; // "Frau"
+echo $faker->salutationMale; // "Herr"
+echo $faker->salutationFemale; // "Frau"
+echo $faker->name; // "Herr Dr. Thomas Müller"
```

Comment:
please inverse the two: `name` should give "Thomas Müller", and `formalName` should give "Herr Dr. Thomas Müller"

Reply:
I think you have misread this one. I did not change the functionality of name. I just removed the not common titles. The new one I have created is name**In**formal. To get a name without titles.